lang/en.json

```diff
@@ -143,7 +143,8 @@
+    "whoWouldYouLikeToBeTheAttorneysDetailsContentPA": "<p class=\"govuk-body\">Your attorneys are the people who will be able to help you make decisions about your money, property and finances, if you ever want or need their help in the future.</p><p class=\"govuk-body\">It’s important that you choose people you trust to respect your wishes and act in your best interests. Your attorneys could be your spouse, family members, close friends or a professional, like a solicitor.</p><p class=\"govuk-body\">You can have more than one attorney. Most people have between 1 and 3 attorneys.</p><p class=\"govuk-body\">If you do choose to have more than one attorney, you should choose people who can work well together.</p><p class=\"govuk-body\">Your attorneys must be aged 18 or over. An attorney must not be bankrupt or have a debt relief order. <a href=\"#\" class=\"govuk-link\">Find out more about attorneys</a>.</p><p class=\"govuk-body\">You’ll be asked if you would like to add <a href=\"#\" class=\"govuk-link\">replacement attorneys</a> in a separate question. They step in if your original attorneys are unable or unwilling to continue.</p>",
+    "whoWouldYouLikeToBeTheAttorneysDetailsContentPW": "<p class=\"govuk-body\">Your attorneys are the people who will be able to help you make decisions about your personal welfare, if you ever need their help in the future.</p><p class=\"govuk-body\">It’s important that you choose people you trust to respect your wishes and act in your best interests. Your attorneys could be your spouse, family members, close friends or a professional, like a solicitor.</p><p class=\"govuk-body\">You can have more than one attorney. Most people have between 1 and 3 attorneys.</p><p class=\"govuk-body\">If you do choose to have more than one attorney, you should choose people who can work well together.</p><p class=\"govuk-body\">Your attorneys must be aged 18 or over. <a href=\"#\" class=\"govuk-link\">Find out more about attorneys</a>.</p><p class=\"govuk-body\">You’ll be asked if you would like to add <a href=\"#\" class=\"govuk-link\">replacement attorneys</a> in a separate question. They step in if your original attorneys are unable or unwilling to continue.</p>",
```

Comment:
"if you ever (need->want) their help" in the first line

Reply:
I think other bits of the content have drifted too, so needs checking what it should be. For example the "You can have more than one attorney..." line in the ticket